$OpenBSD: patch-daemon_data_gnome-keyring_schemas_in,v 1.1 2009/01/29 18:15:20 ajacoutot Exp $

Disable gnome-keyring SSH agent by default. As this is a GConf entry, it
can be enabled easily for who needs it.

--- daemon/data/gnome-keyring.schemas.in.orig	Thu Jan 29 18:21:14 2009
+++ daemon/data/gnome-keyring.schemas.in	Thu Jan 29 18:21:39 2009
@@ -5,7 +5,7 @@
 			<applyto>/apps/gnome-keyring/daemon-components/ssh</applyto>
 			<owner>gnome-keyring</owner>
 			<type>bool</type>
-			<default>TRUE</default>
+			<default>FALSE</default>
 			<locale name="C">
 				<short>Whether the gnome-keyring SSH agent is enabled.</short>
 				<long>This option enables the SSH agent in the gnome-keyring 
